app/controllers/films_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FilmsController < ApplicationController
```

Comment:
`include ShowsHelper` that way you don't have to do `helpers.get_csv_data`, you just do`get_csv_data`